pkgs/by-name/pa/pappl/package.nix

```
@@ -0,0 +1,74 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  avahi,
+  cups,
+  gnutls,
+  libjpeg,
+  libpng,
+  libusb1,
+  pkg-config,
+  withPAMSupport ? true,
+  pam,
+  zlib,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pappl";
+  version = "1.4.7";
+
+  src = fetchFromGitHub {
+    owner = "michaelrsweet";
+    repo = "pappl";
+    rev = "refs/tagsv${version}";
```

```suggestion
    rev = "refs/tags/v${version}";
```

likely the cause of the ofborg 404 error

---

Whoops.

---

Fixed, thanks for catching that.

Though I had to squash it into the by-name commit, not very fond of that but it's the only way of *not* redoing the entire PR I think.

---

in the future it's easier if you start with the reformat commit